aws-http-nio-client-netty/pom.xml

```diff
@@ -48,6 +48,10 @@
             <artifactId>netty-handler</artifactId>
         </dependency>
         <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport-native-epoll</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
I don't mind adding it if there are customer requests but I decided to leave it out for now.